From 2bc8967564fdae46045545f8b98a3160dc916b3d Mon Sep 17 00:00:00 2001
From: Mingqiang Chi <mingqiang.chi@intel.com>
Date: Thu, 10 Jan 2019 15:00:41 +0800
Subject: [PATCH 1216/1334] kernel: Sync the common header file

Change the type from uint8_t to uint32_t for interrupt pin


Change-Id: Ifcd64388a99c810fa30f0270b087c4c6e3d1af07
Signed-off-by: Mingqiang Chi <mingqiang.chi@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
---
 include/linux/vhm/acrn_hv_defs.h | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/include/linux/vhm/acrn_hv_defs.h b/include/linux/vhm/acrn_hv_defs.h
index bb60888..030aed2 100644
--- a/include/linux/vhm/acrn_hv_defs.h
+++ b/include/linux/vhm/acrn_hv_defs.h
@@ -232,12 +232,10 @@ struct hc_ptdev_irq {
 	uint16_t phys_bdf;	/* IN: Device physical BDF# */
 	union {
 		struct {
-			uint8_t virt_pin;	/* IN: virtual IOAPIC pin */
-			uint8_t reserved0[3];	/* Reserved */
-			uint8_t phys_pin;	/* IN: physical IOAPIC pin */
-			uint8_t reserved1[3];	/* Reserved */
+			uint32_t virt_pin;	/* IN: virtual IOAPIC pin */
+			uint32_t phys_pin;	/* IN: physical IOAPIC pin */
 			bool pic_pin;		/* IN: pin from PIC? */
-			uint8_t reserved2[3];	/* Reserved */
+			uint8_t reserved[3];	/* Reserved */
 		} intx;
 		struct {
 			/* IN: vector count of MSI/MSIX */
-- 
2.7.4

